chart/kubeapps/templates/mongodb-secret-bootstrap-rbac.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if .Values.rbac.create -}}
```

Comment:
we should rename the filenames to match the naming, i.e. mongodb-jobs-bootstrap-rbac here